package.json

```diff
@@ -60,6 +60,7 @@
     "redux-logger": "3.0.6",
     "redux-thunk": "2.3.0",
     "sdp-transform": "2.4.1",
+    "simplebar": "3.0.0-beta.5",
```

Comment:
Beta?

Reply:
Yes, this has performance improvements over the latest stable version.
https://github.com/Grsmto/simplebar/releases/tag/simplebar%403.0.0-beta.0